test/integration/short/ssl-tests.js

```diff
@@ -28,6 +29,15 @@ describe('Client', function () {
           helper.finish(client, done)();
         });
       });
+      it('should callback in error when rejecting unauthorized', function (done) {
```

Comment:
We expect this to fail since C* is using a self-signed certificate with CCM is that right?

Reply:
yep :)